Test/default.cpp

```diff
@@ -0,0 +1,54 @@
+/* default.cpp verifies the Teensy is connected for NanoSAM II tests. 
+ * Usage:
+ *  run this script through Teensyduino to power the NS2 payload
+ *
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+ 
+ // I think I'm writing this as Arduino code not c++? 
```

Comment:
Arduino code is C++, just with the inclusion of the Arduino library and the caveat that not all C++ libraries work on our hardware.